src/components/molecules/NewsFlashFilterPanel.tsx

```diff
@@ -31,41 +40,50 @@ const useStyles = makeStyles({
     justifyContent: 'space-evenly',
   },
 });
+
 export const NewsFlashFilterPanel: React.FC = () => {
   const classes = useStyles();
   const store: RootStore = useStore();
+  const [activeFilter, setActiveFilter] = useState(SourceFilterEnum.all);
+
+  const isClicked = (sourceType: SourceFilterEnum) =>
+    activeFilter === sourceType ? classes.buttonClicked : classes.button;
 
   return (
     // will be convert to filterButtonsMap instead of code duplicate for the filter button
     <Box className={classes.container}>
       <AnyWayButton
-        className={classes.button}
+        className={isClicked(SourceFilterEnum.all)}
```

Comment:
`isClicked` here is used to determine the color
either give it an appropriate name or put the logic in the `JSX`
```
className={activeFilter === SourceFilterEnum.all ? classes.buttonClicked : classes.button}
```